src/lib/y2network/interface_type.rb

```diff
@@ -149,5 +149,7 @@ def method_missing(method_name, *arguments, &block)
     IRDA = new(N_("Infrared"), "irda")
     # Loopback
     LO = new(N_("Loopback"), "lo")
+    # Not supported interfaces
+    UNSUPPORTED = new(N_("Unsupported"), "unsupported")
```

Comment:
Why not `UNKNOWN`? After all, that's the word you have used in the PR's description.

Reply:
I was not sure, because the type is recognized by hwinfo, so, it is more a known one but unsupported ... so, not have a strong opinion about which one should be used.

Reply:
And you have already realized I used **unknown** instead of unsupported in the description as I was already doubting the term to use

Reply:
well, unknown and unsupported have slightly different meaning. We can e.g. know the type but do not support its configuration ;-)